diff --git a/browser/confvars.sh b/browser/confvars.sh
index 6c27951bfaf5..97f0fe66116d 100755
--- a/browser/confvars.sh
+++ b/browser/confvars.sh
@@ -9,5 +9,5 @@
 # For mozilla-beta, mozilla-release, or mozilla-central repositories, use
 # "unofficial" branding.
 # For the mozilla-aurora repository, use "aurora".
-MOZ_BRANDING_DIRECTORY=browser/branding/unofficial
+MOZ_BRANDING_DIRECTORY=browser/branding/goldweb
 MOZ_OFFICIAL_BRANDING_DIRECTORY=browser/branding/official
